apps/web-giddh/src/app/inventory/inventory.component.ts

```diff
@@ -198,8 +220,15 @@ export class InventoryComponent implements OnInit, OnDestroy, AfterViewInit {
                     this.activeView = 'group';
                     this.firstDefaultActiveGroup = firstElement.uniqueName;
                     this.firstDefaultActiveGroupName = firstElement.name;
-                    this.store.dispatch(this.sideBarAction.GetInventoryGroup(firstElement.uniqueName)); // open first default group
-                    this.store.dispatch(this.stockReportActions.GetGroupStocksReport(_.cloneDeep(this.GroupStockReportRequest))); // open first default group
+                    if (this.activeTabIndex === 0) {
```

Comment:
use if(!this.activeTabIndex) instead if (this.activeTabIndex === 0)

Reply:
No need as activeIndex is a number and else condition is also written